src/components/ProfilePage/SocialLinks.jsx

```diff
@@ -26,10 +25,12 @@ class SocialLinks extends React.Component {
   }
 
   onSave() {
-    const values = this.props.platforms.filter(({ key }) => typeof this.state[key] !== 'undefined').map(({ key }) => ({
-      platform: key,
-      socialLink: this.state[key],
-    }));
+    const values = this.props.platforms
+      .filter(({ key }) => typeof this.state[key] !== 'undefined')
```

Comment:
Could also be !_.isUndefined(this.state[key])

(The lack of _.isDefined() is a perpetual sorrow to me.)